vendor.mod

```diff
@@ -80,8 +80,8 @@ require (
 	github.com/pelletier/go-toml v1.9.5
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.14.0
-	github.com/rootless-containers/rootlesskit v1.1.0
-	github.com/sirupsen/logrus v1.9.0
+	github.com/rootless-containers/rootlesskit v1.1.1
+	github.com/sirupsen/logrus v1.9.2
```

Comment:
We should update this one to v1.9.3, as v1.9.2 was borked. https://github.com/sirupsen/logrus/releases/tag/v1.9.3

Reply:
solved via https://github.com/moby/moby/pull/45706/commits/57889cebdfee1bd746a59cbf640e0426f24abeed